package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@salesforce/commerce",
-    "version": "234.0.18",
+    "version": "234.0.19",
```

Comment:
Should this be 238 instead of 234?

Reply:
@wei-liu-salesforce I believe @ksiemer will bump this up to 238 at a later date